README.md

```diff
@@ -53,15 +53,13 @@ vars:
 
 ### Passthrough Columns
 
-This package includes all source columns defined in the macros folder. If you want to include custom fields in this package, you can add more columns using our pass-through column variables.
+This package includes all source columns defined in the macros folder. If you want to include custom fields in this package, you can add more columns using our pass-through column variables to your `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
 vars:
   intercom__company_history_pass_through_columns: [company_custom_field_1, company_custom_field_2]
   intercom__contact_history_pass_through_columns: [super_cool_contact_field]
+  intercom__conversation_history_pass_through_columns: [another_custom_field, another_custom_field_2]
```

Comment:
Since you updated the way the passthrough variables are provided, we should also likely update the docs for it to be the updated version. 

Similar to how customers can use the name and alias.
```yml
  intercom__company_history_pass_through_columns: 
    - name: custom_employee_range_c
      alias: employee_range 
    - name: custom_monthly_spend
    - name: custom_paid_subscriber
```

Reply:
Done. I borrowed the format I used in Recharge since that seemed to work.